scripts/stav-covid.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+import requests
+from bs4 import BeautifulSoup
+import datetime
+import re
+import warnings
+import yaml
+
+parser = argparse.ArgumentParser(description="Load hours from St. Olaf's COVID dining hours site")
+parser.add_argument('-o', '--output', type=argparse.FileType('w'), required=True,
+                    help='the file to write the schedule into')
+options = parser.parse_args()
+
+# Fetch the output to a string, `output`.
+url = "https://wp.stolaf.edu/reslife/dining-hours/"
+output = requests.get(url).text
+
+# Now, parse it.
+output = BeautifulSoup(output)
```

Comment:
A nit –– feel free to ignore.

> GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("html.parser"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.
>
> The code that caused this warning is on line 19 of the file s.py. To get rid of this warning, pass the additional argument 'features="html.parser"' to the BeautifulSoup constructor.

```suggestion
output = BeautifulSoup(output, features="html.parser")
```